lib/vanagon/engine/base.rb

```diff
@@ -102,7 +102,7 @@ def retrieve_built_artifact(artifacts_to_fetch, no_packaging)
       def validate_platform
         missing_attrs = []
         @required_attributes.each do |attr|
-          if !@platform.instance_variables.include?("@#{attr}".to_sym) || @platform.instance_variable_get("@#{attr}".to_sym).nil?
+          if !@platform.instance_variables.include?("@#{attr}".to_sym) || @platform.instance_variable_get("@#{attr}".to_sym).nil? #rubocop:disable Lint/SymbolConversion
```

Comment:
We should disable this cop in the .rubocop.yml file rather than here so we have consistent policy and less clutter in the source code.